services/gateway/apis/http-version2/swagger/parameters.json

```diff
@@ -0,0 +1,304 @@
+{
+    "parameters": {
+        "ip": {
+            "name": "ip",
+            "in": "query",
+            "description": "IP of the node or delegate",
+            "type": "string",
+            "format": "ip"
+        },
```

Comment:
Are these all common input params? Can they be associated with peers, for instance?

Reply:
Done!